### ISSUE TITLE
Not possible to select attribute with name status

### ISSUE THREAD

**Issue by janjilek** [orientechnologies/orientjs#187]
When I try to execute query "select status from Gift" via query builder, I receive only following response:
{ '@type': 'd',
    '@rid': { [String: '#-2:99'] cluster: -2, position: 99 },
    '@version': 0 }

As you can see, status is missing. When I try to execute it in console, everything works well. When I try to execute it on OrientDB 2.1.16, everything is fine. The problem is only with driver 2.2 and OrientDB 2.2 (.4)

Regards
Jan

**Comment by maggiolo00** [comment]
do you have an example script?

**Comment by janjilek** [comment]
For query builder? It's really simple, only:
```
db
      .select('status')
      .from('Gift')
      .all()
```
Schema is also really simple:
field status of type number
and not unique index on status

**Comment by maggiolo00** [comment]
can you post the schema also?

and some data thanks

**Comment by maggiolo00** [comment]
Can you run this script against the Orientdb 2.2.4? with  the default database?

```JS
const OrientJS = require('orientjs'),
    server = OrientJS({ host: 'localhost', port: 2424 }),
    db = server.use({ name: 'GratefulDeadConcerts', username: 'admin', password: 'admin' });

db.open().then(function(){

	db.select('name').from("OUSer").all().then(function(res){
		console.log(res);
		server.close();
	});

})
```

**Comment by janjilek** [comment]
Sure, I receive 
```
[ { '@type': 'd',
    name: 'admin',
    '@rid': { [String: '#-2:0'] cluster: -2, position: 0 },
    '@version': 0 },
  { '@type': 'd',
    name: 'reader',
    '@rid': { [String: '#-2:1'] cluster: -2, position: 1 },
    '@version': 0 },
  { '@type': 'd',
    name: 'writer',
    '@rid': { [String: '#-2:2'] cluster: -2, position: 2 },
    '@version': 0 } ]
```
and I am using Orientdb 2.2.4 :-)

**Comment by maggiolo00** [comment]
So the projection seems to work

if you can share your dataset i can inspect it further.

You can send me some example here e.risa@orientdb.com

Thanks

**Comment by janjilek** [comment]
I just sent you dataset to your email.

Thanks

**Comment by janjilek** [comment]
Ok, thank you, i will migrate it. But you should also add this restrictions to the documentation :-) 

Jan

**Comment by janjilek** [comment]
What is the best way to convert from Decimal to Double? Seems to be not possible to change it only in schema

**Comment by maggiolo00** [comment]
hi @janjilek 
it is fixed in the newer version 2.2.2 with decimal support.
Thanks

**Comment by janjilek** [comment]
Great! 
Thank you
